### ISSUE TITLE
[Question] Battlemage armor spells

### ISSUE THREAD

**Issue by ThatOneTraitor** [Mithion/ArsMagica2#1481]
Its mentioned in the compendium that the battlemage armor will cast "protective spells." Could someone tell me what those effects are? i cant seem to get them to activate

**Comment by TheIcyOne** [comment]
I've been looking through the code and have seen no reference to any protective spells relating to the battlemage armour, and I am getting a slight sense of deja vu whilst writing this.

**Comment by ThatOneTraitor** [comment]
I guess maybe it was replaced by armor imbuement as someone else said, but i'll wait for this to be confirmed before closing.

**Comment by TheIcyOne** [comment]
If it wasn't, that actually sounds like a fun feature. Perhaps spells could be "socketed" into mage armour pieces to be autocast on a specific event?

**Comment by ThatOneTraitor** [comment]
yeah, from what i HEARD. the leggings would sometimes give lava resistance, the boots would cast slowfall when your falling from a high place, and if you were in water the helmet gave you water breathing. all of these were put into imbuements.. so im guessing its was moved. for what @TheIcyOne said, it actually sounds fun, but it would have to be expensive.

**Comment by Zieg777** [comment]
They used to do all those things. Those are now in imbuements though. I'm in favor of the Mage/Battlemage doing something at least though. Looking at the code, I don't know that they even increase mana gen without imbuements.

Perhaps something akin to Botania armors? Either complete set grants bonuses (mana regen, more powerful spells, etc) or individual pieces do things.

Complete set means there might be a reason to make the battlemage chestpiece. My default is Earth Armor for that.
